main.go

```diff
@@ -63,17 +65,18 @@ const (
 )
 
 var (
-	globalQuietEnabled   bool
-	globalDebugEnabled   bool
-	globalLoggingEnabled bool
-	globalTrace          string
-	globalJSONEnabled    bool
-	globalConsoleDisplay bool
-	globalErrorsOnly     bool
-	globalStatusCodes    []int
-	globalConnStats      []*ConnStats
-	log2                 *logrus.Logger
-	globalHostBalance    string
+	globalQuietEnabled     bool
+	globalDebugEnabled     bool
+	globalLoggingEnabled   bool
+	globalTrace            string
+	globalJSONEnabled      bool
+	globalConsoleDisplay   bool
+	globalErrorsOnly       bool
+	globalStatusCodes      []int
+	globalConnStatsRWMutex sync.RWMutex
```

Comment:
```suggestion
	globalConnStatsRWMutex sync.Mutex
```
No big need for this to be a read/write mutex if we just grab the value.